frontend/src/components/tests/ItemEditor.test.js

```diff
@@ -1,21 +1,25 @@
-import { render, screen } from "@testing-library/react";
+import { render, screen, waitFor } from "@testing-library/react";
 import "@testing-library/jest-dom";
 import "regenerator-runtime/runtime";
 import React from "react";
-import { Link } from "react-router-dom";
 import ItemEditor from "../ItemEditor.js";
 
-test("renders item editor layer", (done) => {
-  //render(<ItemEditor type="TopLevelNormativeGoal" id="0"/>);
-  //try {
-  //  const textElement = screen.getByText(/TopLevelNormativeGoal/i);
-  //  expect(textElement).toBeInTheDocument();
-  //  done()
-  //} catch (error) {
-  //  done(error)
-  // }
-  //const textElement = screen.getByText(/TopLevelNormativeGoal/i);
-  //expect(textElement).toBeInTheDocument();
-  done();
-  expect(true);
+global.fetch = jest.fn(() =>
+  Promise.resolve({
+    json: () =>
+      Promise.resolve({
+        id: 1,
+        name: "Test goal",
+        short_description: "short",
+        long_description: "long",
+        keywords: "key",
+      }),
+  })
+);
+
+test("renders item editor layer", async () => {
+  render(<ItemEditor type="TopLevelNormativeGoal" id="0" />);
```

Comment:
I'm a bit confused about using `id="0"` here, but the mock fetch returning `id: 1`.

Reply:
I'm not sure this makes any difference - may as well change to make them consistent, but I think the fetch call will return whatever is in the mock, regardless of what "id" was in the props.